README.md

```diff
@@ -139,24 +204,8 @@ sql commands that can be run in sequence to create the base state we want.
 # import bancho.py's mysql structure to our new db
 # this runs the contents of the file as sql commands.
 mysql -u YOUR_DB_USER -p YOUR_DB_NAME < migrations/base.sql
-```
-
-## creating a ssl certificate (to allow https traffic)
-```sh
-# you'll need to change:
-# - YOUR_EMAIL_ADDRESS
-# - YOUR_DOMAIN
 
-# generate an ssl certificate for your domain
-sudo certbot certonly \
-    --manual \
-    --preferred-challenges=dns \
-    --email YOUR_EMAIL_ADDRESS \
-    --server https://acme-v02.api.letsencrypt.org/directory \
-    --agree-tos \
-    -d *.YOUR_DOMAIN
 ```
-
```

Comment:
I think there's still value in keeping these instructions if users wish to do SSL themselves. Perhaps this could be split between a "cloudflare flex" setup, and a manual ssl setup?

Reply:
they are below